packages/cli/tsconfig.build.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.json",
   "include": ["src"],
+  "exclude": ["src/**/__tests__/**/*.ts"],
```

Comment:
Probably dont need anything in tests right? Not just skipping .ts but... everything in __test__ ?

Reply:
OOOOh ignore that its only going to try and compile .ts. right.

Reply:
I tried. It just does not work unfortunately, and the last thing I want to do is check out the source code of TypeScript _again_.